internal/pkg/template/diff/write.go

```diff
@@ -38,9 +40,12 @@ func (s *treeWriter) write() error {
 
 func (s *treeWriter) writeTree(node diffNode, indent int) error {
 	var formatter formatter
-	switch node.(type) {
-	// case *unchangedNode:
-	// TODO(lou1425926): handle unchanged. 
+	switch node := node.(type) {
+	case *unchangedNode:
+		content := fmt.Sprintf("(%s)", english.Plural(node.unchangedCount(), "unchanged item", "unchanged items"))
```

Comment:
```suggestion
		content := fmt.Sprintf("(%d)", english.Plural(node.unchangedCount(), "unchanged item", "unchanged items"))
```

Reply:
`english.Plural` returns a string that looks like `"1 unchanged item"` or `"x unchanged items"`!

Reply:
oooh i see now! cool 👍